app/controllers/reports_controller.rb

```diff
@@ -74,6 +74,16 @@ def destroy
   end
 
   def send_report_csv
+    # override DEFAULT_PARAMS of report_helper.rb to get raw data
```

Comment:
a bit hacky. probably should set a flag in geneate_report_csv to turn off the thresholdings. genus_search would need it too. @boris-dimitrov. At least at a TODO here.

Reply:
ok